SimPEG/DataMisfit.py

```diff
@@ -47,12 +47,12 @@ def evalDeriv(self, m, u=None):
 
 
     @Utils.timeIt
-    def eval2Deriv(self, m, v, u=None):
-        """eval2Deriv(m, v, u=None)
+    def eval2Deriv(self, m, v, f=None):
+        """eval2Deriv(m, v, f=None)
 
             :param numpy.array m: geophysical model
             :param numpy.array v: vector to multiply
-            :param numpy.array u: fields
+            :param Fields f: fields
```

Comment:
does `Fields` show up in the docs? It might be better to be `SimPEG....Fields`? (where ever it actually is)